api/accesslog.proto

```diff
@@ -169,3 +169,30 @@ message EnvoyAccessLog {
   google.protobuf.UInt32Value tls_cipher_suite = 30;
 }
 
+// Stream message for the StreamAccessLogs API. Envoy will open a stream to the server and stream
+// access logs without ever expecting a response.
+message EnvoyAccessLogMessage {
+  // The node sending the access log messages over the stream. Note that the server should only
+  // expect node to be populated on the first message in the stream. It will not be populated in
+  // subsequent messages. This is a performance optimization.
+  Node node = 1;
+  // The individual access log message.
+  EnvoyAccessLog log = 2;
```

Comment:
I don't think there is a huge benefit in making this a `repeated`, the byte saving on the wire is probably tiny compared to the size of even a single log message, but throwing it out there for consideration.

Reply:
I don't have a strong opinion this. I don't see Envoy every sending more than one log message at a time but we could put it in to future proof it if we wanted to buffer a few logs before sending for example.